src/rules/map-keys-always-quoted/README.md

```diff
@@ -0,0 +1,26 @@
+# map-keys-always-quoted
```

Comment:
Instead of `map-keys-always-quoted`, this rule could be called something like `map-keys-quotes` with options `always`/`never`:
https://github.com/stylelint/stylelint/blob/master/docs/user-guide/about-rules.md#about-rule-names

Reply:
There is a similar rule in stylelint for `font-family` quotes:
https://github.com/stylelint/stylelint/blob/master/lib/rules/font-family-name-quotes/README.md

Reply:
I'm not sure if an always/never split would be necessary in this case.

It seems like you'd never want a situation where every key in your map is unquoted.

Reply:
Right, in that case the only option could be `always`, like in the font-family rule.

Reply:
Is there a difference between using a "always" flag instead of a true/false flag?

Reply:
IMO there are at least two things why it would make sense to use `always` instead of `true`.

1) Consistency. The other `-quotes` rules in stylelint use the `always`/`never` naming. (e.g. [font-family-name-quotes](https://github.com/stylelint/stylelint/blob/master/lib/rules/font-family-name-quotes/README.md) and [function-url-quotes](https://github.com/stylelint/stylelint/blob/master/lib/rules/function-url-quotes/README.md))
2) Ability to add more options in the future. If we use `always`, then we can add `always-*`, `never-*`, or `never` options if they are needed. Like in the `font-family-name-quotes` rule: https://github.com/stylelint/stylelint/blob/master/lib/rules/font-family-name-quotes/README.md#options

Reply:
That makes so much sense. Changes made!